EXPLAINER.md

```diff
@@ -1,118 +1,108 @@
-- report back a new trace ID in case the callee ignored the existing headers but started a new trace
-- let proxies delegate sampling decisions to the caller
-- allow tail-based sampling, where a sampling decision is deferred until a request is completed and all information pertinent to the sampling decision is known
-- to correlate the initial page load of a browser and all subsequent requests to a server side trace
+### Non Goals
+- Identification of individual users or user sessions.
+- Definition of (programming) language specific APIs for performance data collection.
+- Performance data analysis techniques or algorithms.
+
+### Concepts
+
+#### Distributed Trace
+A distributed trace is a set of events, triggered as a result of a single logical operation, consolidated across various components of an application. A distributed trace contains events that cross process, network and security boundaries. A distributed trace may be initiated when someone presses a button to start an action on a website - in this case, the trace will represent calls made between the downstream services that handled the chain of requests initiated by this button being pressed.
 
-### Baggage
+##### Headers
 
-The baggage header is used to propagate properties not defined in trace parent.
+###### Traceparent
 
-Common use cases are
+The traceparent header is a request header which contains a trace id, a parent span id, and trace flags. The traceflags includes information such as if a particular request is sampled. When a tracing system transmits a span to the tracing backend, it includes the Trace ID, the span ID, and the parent span ID (if one exists). The tracing backend uses this information to construct the directed acyclic graph which represents the trace. In a system with a single tracing platform, it is the only header necessary to complete a distributed trace.
 
-* Defining an application specific context such as a member status on a trace
-* Adding a marker for a/b testing
-* Passing the callers name to the next component
+###### Tracestate
 
+The tracestate header is a request header which allows a tracing platform to transmit platform-specific information, even through systems that are traced using other tracing platforms. Each tracing platform uses a single key, and the value is treated as an opaque token by other tracing platforms. Each tracing platform is required to forward tracestate keys and values.
 
-## Examples
+### Examples
 
-### Context loss using a proprietary header
+#### Context loss using a proprietary header
 
 1. Service A calls Service B through an API gateway.
 2. The tracing system uses a proprietary header to propagate its trace ID.
-3. The API gateway is not configured to propagate the proprietary header
-4. Service C does not receive a trace ID and context is lost
+3. The API gateway is not configured to propagate the proprietary header.
+4. Service C does not receive a trace ID and context is lost.
```

Comment:
I know you only added a `.` at the end here, but wouldn't that need to read 

```suggestion
4. Service B does not receive a trace ID and context is lost.
```

to make sense? The example is A -> API gateway -> B, right?